### ISSUE TITLE
Object detection ssd_mobilenet_v2 evaluation on COCO

### ISSUE THREAD

**Issue by snownus** [tensorflow/models#5106]
Get the release pretrained model ssd_mobilenet_v2 evaluation on COCO, how to evaluate? Seems the official code only provides evaluation on Open Image Challenges 2018. I have tried model_main.py, but doesn't work.
python model_main.py
--pipeline_config_path=experiments/coco/configs/pipeline.config
--checkpoint_dir=models/ssd_mobilenet_v2/
--model_dir=output/experiments/coco/ssd_mobilenet_v2/
--run_once=True
--alsologtostderr

@pkulzc

**Comment by pkulzc** [comment]
Your command looks right, what message did you get ?

**Comment by pkulzc** [comment]
Eval input path should be in your config file.

**Comment by snownus** [comment]
@pkulzc , Yes, currently I use COCO2014 to test, the performance is 24.1, while the official one is 22. So I want to know which COCO dataset the your side use?  Is it COCO2017?

**Comment by netanel-s** [comment]
I believe the difference is that the official metric is on COCO's test set, while I assume you tested on COCO's val set.

**Comment by snownus** [comment]
@netanel-s , ok. Thanks.

One more question, that, does the SSD-MobilNetV1 and SSD-MobileNetV2 has the same preprocessing as https://github.com/tensorflow/models/blob/master/research/object_detection/object_detection_tutorial.ipynb. I run SSD-MobileNet-V2 and found that the performance is worse than SSD-MobileNet-V1. 

I want to extract some intermediate layers data, so the preprocessing is very important to me.

**Comment by netanel-s** [comment]
Is the claim that the performance is worse based only on the given test images of the tutorial?
These two examples might not be representative of the models' performances.
Try evaluating both on COCO's val set or at check the results on much more examples.

**Comment by snownus** [comment]
@netanel-s , the issue is addressed. Thanks.
